diff --git a/chainregistry.go b/chainregistry.go
index adbc906d..c1a562b7 100644
--- a/chainregistry.go
+++ b/chainregistry.go
@@ -55,7 +55,7 @@ const (
 
 	// btcToLtcConversionRate is a fixed ratio used in order to scale up
 	// payments when running on the Litecoin chain.
-	btcToLtcConversionRate = 60
+	btcToLtcConversionRate = 90
 )
 
 // defaultBtcChannelConstraints is the default set of channel constraints that are
diff --git a/fundingmanager.go b/fundingmanager.go
index bc272312..8e27f5f4 100644
--- a/fundingmanager.go
+++ b/fundingmanager.go
@@ -62,7 +62,7 @@ const (
 	// Protocol. This limit is defined in BOLT-0002, and serves as an
 	// initial precautionary limit while implementations are battle tested
 	// in the real world.
-	maxBtcFundingAmount = btcutil.Amount(1<<24) - 1
+	maxBtcFundingAmount = btcutil.Amount(1<<32) - 1
 
 	// maxLtcFundingAmount is a soft-limit of the maximum channel size
 	// currently accepted on the Litecoin chain within the Lightning
diff --git a/rpcserver.go b/rpcserver.go
index bef358f3..459734f3 100644
--- a/rpcserver.go
+++ b/rpcserver.go
@@ -59,11 +59,11 @@ import (
 const (
 	// maxBtcPaymentMSat is the maximum allowed Bitcoin payment currently
 	// permitted as defined in BOLT-0002.
-	maxBtcPaymentMSat = lnwire.MilliSatoshi(math.MaxUint32)
+	maxBtcPaymentMSat = lnwire.MilliSatoshi(math.MaxUint32) * 64
 
 	// maxLtcPaymentMSat is the maximum allowed Litecoin payment currently
 	// permitted.
-	maxLtcPaymentMSat = lnwire.MilliSatoshi(math.MaxUint32) *
+	maxLtcPaymentMSat = lnwire.MilliSatoshi(math.MaxUint32) * 64 *
 		btcToLtcConversionRate
 )
 

