diff --git a/chainregistry.go b/chainregistry.go
index adbc906d..c1a562b7 100644
--- a/chainregistry.go
+++ b/chainregistry.go
@@ -55,7 +55,7 @@ const (
 
 	// btcToLtcConversionRate is a fixed ratio used in order to scale up
 	// payments when running on the Litecoin chain.
-	btcToLtcConversionRate = 60
+	btcToLtcConversionRate = 90
 )
 
 // defaultBtcChannelConstraints is the default set of channel constraints that are
